pages/blog/software-development-workflow.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: "Software Development Workflow with the Grouparoo UI"
```

Comment:
Per our SEO suggesting <60 chars for page titles, we may want something like "Development Workflow with the Grouparoo UI" instead (58 characters once we add ": Grouparoo Blog")

Reply:
"Development workflow for Reverse ETL" is probably my favorite.